lib/opensusebasetest.pm

```diff
@@ -398,6 +381,26 @@ sub wait_grub {
     mutex_wait 'support_server_ready' if get_var('USE_SUPPORT_SERVER');
 }
 
+=head2 wait_grub_to_boot_on_local_disk
```

Comment:
Seems that you've missed putting the description and closing section.
It should be like:
```perl
=head2 wait_grub_to_boot_on_local_disk

    wait_grub_to_boot_on_local_disk

<Description goes here>

=cut
```

Reply:
Updated, please have a look.